src/main/java/frc/robot/subsystems/Module.java

```diff
@@ -95,16 +130,36 @@ private void setAngle(SwerveModuleState desiredState) {
       stop();
       return;
     }
-
     m_angleMotor.set(ControlMode.Position, Conversions.degreesToFalcon(desiredState.angle.getDegrees(), DriveConstants.kAngleGearRatio));
+    if(Constants.kLogging){
+      double position = Conversions.falconToDegrees(m_angleMotor.getSelectedSensorPosition(), 
+        DriveConstants.kAngleGearRatio);
+      LogManager.addDoubleArray("Swerve/Modules/"+m_moduleAbbr+"/SteerPosition",
+        new double[]{
+          desiredState.angle.getDegrees(), 
+          position,
+          position-desiredState.angle.getDegrees(),
+          Conversions.falconToDegrees(m_angleMotor.getSelectedSensorVelocity(), 
+            DriveConstants.kAngleGearRatio)
+        }
+      );
+      LogManager.addDoubleArray("Swerve/Modules/"+m_moduleAbbr+"/SteerPower",
+        new double[]{
+          m_angleMotor.getMotorOutputVoltage(), 
+          m_angleMotor.getStatorCurrent(),
+        }
+      );
```

Comment:
desisered angle and position are already logged in drivetrain. The other things being logges should not be in arrays but individual logs so it is clear to someone looking at the logs what they represent